docs/guides/use-shared-project-for-organization-wide-collaboration.md

```diff
@@ -92,22 +94,21 @@ You can open the process landscape view by clicking **View landscape** from any
 
 2. **Search**: Press `Ctrl+F` or `⌘+F` to search. Enter the name or identifier of a BPMN file to quickly find, highlight, and jump to the corresponding node.
    ![landscape search](../components/modeler/web-modeler/img/process-landscape-search.png)
-3. **Highlight Paths:** Click on a node or connection to highlight the entire chain of related connections.
+3. **Highlight paths:** Click on a node or connection to highlight the entire chain of related connections.
    ![landscape selected node connections](../components/modeler/web-modeler/img/process-landscape-connection.png)
 
-### README Documentation
+### README documentation
 
 To access the associated README file for a process within the shared project:
 
 1. Open the process landscape view.
 2. Click on a node in the landscape.
 3. The README file (if one is associated) is displayed in the sidebar, providing a high-level overview of the process.
 
-## Reuse resources from a shared project
+## Next steps
 
-Users can reuse published resources by copying them into their own projects:
+Organization owners and administrators can connect their Web Modeler process applications to GitHub, allowing users to keep their Web Modeler, Desktop Modeler, and official version control projects synced.
```

Comment:
This would be a great spot to add a diagram with proper visual design similar to the one I shared in the [comment](https://github.com/camunda/product-hub/issues/2611#issuecomment-2761525582)
1. Sync process application to feature branch of subfolder in a repository.
2. Merge feature branch to main.
3. Begin CI/CD pipeline [NOTE: here's a good place to link to the CI/CD page]
4. Sync main branch to shared project as part of the CI/CD pipeline

This way the process landscape is kept up-to-date with all changes in production.